test/rules/assertions/noUndefinedTypes.js

```diff
@@ -396,17 +400,19 @@ export default {
       settings: {
         jsdoc: {
           structuredTags: {
-            namepathDefiner: {
+            namepathReferencing: {
               name: 'namepath-referencing',
             },
           },
         },
       },
     },
     {
+      // An unknown tag without an namepath contents declared via settings,
+      // defaults to not having an impact on the type.
       code: `
       /**
-       * @namepathDefiner SomeType
+       * @namepathMentioning SomeType
```

Comment:
Since this doesn't define nor reference types, I feel "mention" is more describing.